docs/root/faq/windows/win_run_as_service.rst

```diff
@@ -0,0 +1,47 @@
+Can I run Envoy on Windows under SCM?
+=====================================
+
+.. note::
+
+    This feature is still in Experimental state.
+
+
+ You can start Envoy as Windows Service that is managed under `Windows Service Control Manager <https://docs.microsoft.com/en-us/windows/win32/services/using-services/>`_.
```

Comment:
i think this needs to be dedented - the rest of the page is appearing inside of the note above